hooks/post-turn-quality-stop-hook.py

```
@@ -0,0 +1,937 @@
+    ----------
+    cmd
+        Command and arguments to run.
+    cwd
+        Working directory for the subprocess.
+
+    Returns
+    -------
+    subprocess.CompletedProcess[str]
+        Completed process with captured output.
+    """
+    return subprocess.run(  # FIXME: noqa: S603 - command and args are controlled (no shell, no user-supplied command strings).
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)